index.html

```diff
@@ -671,16 +671,36 @@ <h4>OAuth 2.0</h4>
     <section>
       <h3>Service Instances</h3>
       <p>
-Service configuration is determined by administrators on per instance basis.
-Instance configuration can include, but is not limited to, credential format, key options, credential status mechanisms, and/or credential templates.
-Administrators are expected to configure service instances such that `options` included in client requests cannot result in incorrect action or problematic responses by the service.
+The APIs defined in this specification presume that they are attached to a
+specific instance of some kind with an associated configuration that has been
+put in place by a systems administrator. When a client calls an endpoint on a
+particular instance, the instance uses the configuration and options provided by
+the client to execute the action.
       </p>
       <p>
-A coordinator instance can have access to multiple service instances in order to support different use cases.
-Runtime discovery of service instance configuration is not supported by the VC API as services are expected to be known by the coordinator at the time of coordinator deployment.
+For example, the `/credentials/issue` endpoint is expected to be at the end of a
+longer URL such as `/instances/12345/credentials/issue`. In this case, it is the
+instance that is configured to know which cryptographic key to use for issuance,
+whether or not a status list is involved, the type of credential to issue, the
+credential format, and what additional options are possible on the endpoint.
+      </p>
+      <p>
+Software clients that call a particular instance might not have the capability
+to configure an instance, or be aware of the setup that the administrator did on
+the instance. Administration endpoints for configuring instances could be
+provided by implementations but are not necessary to expose as HTTP APIs;
+configuration can also be done through configuration files or graphical
+interfaces.
+      </p>
+      <p class="note"
+         title="Use cases can use multiple instances for complex flows">
+A coordinator instance can have access to multiple service instances in order to
+support different use cases. Runtime discovery of service instance configuration
+is not supported by the VC API as services are expected to be known by the
+coordinator at the time of coordinator deployment.
```

Comment:
IMO, the current title is too narrow. In my mind, a single coordinator could support multiple use cases and so might need multiple service instances, even the each use case flow is "simple". So having the title suggest that the main use for multiple instances is "complex" flows is too narrow.
```suggestion
      <p class="note"
         title="A coordinator can use multiple service instances">
A coordinator instance can have access to multiple service instances in order to
support different use cases or a use case with complex flows. Runtime discovery of service instance configuration
is not supported by the VC API as services are expected to be known by the
coordinator at the time of coordinator deployment.
```

Reply:
We might also want to say that "Using multiple service instances, such as different workflows, can also be used to improve a deployment's security profile by limiting the privileges that a coordinator needs to operate. One way this can be done is by restricting the inputs a coordinator can provide when creating an exchange to just a few variables in a workflow-specific template." Or something along these lines.